### PR TITLE
added Navigation and Slider component

### PR DESCRIPTION
**Pull request by Nikhil-Nirmal**
https://zest-ui-test-deployment.netlify.app/

### CODE REVIEW

File: components/Extras/Extra-Components/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+.navigation ul {
+    background-color: rgb(77, 76, 76);
+}
+
+.navigation li a:hover {
+    color: rgb(167, 214, 255);
+    text-decoration: underline;
+}
+
+.navigation-vertical li a{
```
Comment (ScoobyDubeyDo) [inline components/Extras/Extra-Components/Navigation.css]: Avoid grabbing elements by their name. Use class in place of it.

**Review by ScoobyDubeyDo — CHANGES_REQUESTED** [review]

File: components/Extras/Extra-Components/Navigation.css (hunk 1 above)
Comment (Nikhil-Nirmal) [reply components/Extras/Extra-Components/Navigation.css]: OKAY WILL DO SO

Comment (Nikhil-Nirmal) [reply components/Extras/Extra-Components/Navigation.css]: Changes are done

File: components/Extras/Extra-Components/Navigation.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    text-decoration: none;
+    color: white;
+}
+
+.navigation .nav-list {
+    display: inline-block;
+    margin: 0.5rem 1rem;
+}
+.navigation .nav-ul {
+    background-color: rgb(77, 76, 76);
```
Comment (md-amir-gauhar) [inline components/Extras/Extra-Components/Navigation.css:11]: Use variables for colors

File: components/Extras/Extra-Components/Navigation.html [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+        <ul >
+            <li><a href="../Extra-Components/Modal.html">&lt;&emsp;Modal</a></li>
+            <li><a href="../Extra-Components/Slider.html">Slider&emsp;&gt;</a></li>
+
+        </ul>
+    </footer>
+
+    <script>
+        function openNav() {
+          document.getElementById("mySidenav").style.width = "250px";
```
Comment (md-amir-gauhar) [inline components/Extras/Extra-Components/Navigation.html:259]: Try to write js in external js file

**Review by md-amir-gauhar — APPROVED** [review]

**Review by ScoobyDubeyDo — APPROVED** [review]

**Pull request merged**